_dashboards/im-dashboards/index.md

```diff
@@ -11,6 +11,22 @@ redirect_from:
 Introduced 2.5
 {: .label .label-purple }
 
-Previously, users relied on REST APIs or YAML configurations for basic administrative operations and interventions. This release takes the first step toward a unified administration panel in OpenSearch Dashboards with the launch of several index management UI enhancements. The new interface provides a more user-friendly way to run common indexing and data stream operations. Now you can perform create, read, update, and delete (CRUD) and mapping operations for indexes, index templates, and aliases through the UI. Additionally, you can open, close, reindex, shrink, and split indexes. The UI runs index status and data validation before submitting requests and lets you compare changes with previously saved settings before making updates.
+The Index Management interface in OpenSearch Dashboards provides a unified solution for managing common indexing and data stream operations. The interface allows you to perform create, read, update, and delete (CRUD) and mapping operations for indexes, index templates, and aliases, instead of using REST APIs or YAML configurations for basic administrative operations and interventions, along with other operations such as open, close, reindex, shrink, and split indexes. The interface also provides you with capabilities to run index status and data validation before submitting requests and compare changes with previously saved settings before making updates.
 
-<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/admin-UI-preview.gif" alt="Index management demo gif">{: .img-fluid}
\ No newline at end of file
+## Get started with index management using Dashboards
+
+**Step 1: Open Index Management** Once you're in OpenSearch Dashboards, select **Index Management** under the Management main menu. Then, select  **Indices**.
+
+**Step 2: View indexes** In the Indices interface you will see a list of existing indexes in your OpenSearch cluster. The list provides information such as index name, health state, document count, index size, and other relevant details.  
+
+**Step 3: Create an index** To create a new index, select the **Create index** button in the upper right corner. You will be prompted to enter the index name and configure the index settings, such as number of shards and replicas. Fill in the required information and select **Create** to create the index.
+
+**Step 4: Delete an index** To delete an index, locate the index and select the checkbox next to it. Then, select the **Actions** button and choose **Delete** from the dropdown list. Use caution when deleting indexes, as this action is irreversible. 
```

Comment:
```suggestion
**Step 4: Delete an index** To delete an index, locate the index and select the checkbox next to it. Then select the **Actions** button and choose **Delete** from the dropdown list. Use caution when deleting indexes because this action is irreversible. 
```